src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -636,13 +641,16 @@ ReservedHeapSpace::ReservedHeapSpace(size_t size, size_t alignment, size_t page_
   guarantee(is_aligned(size, alignment), "set by caller");
 
   if (UseCompressedOops) {
+    NOT_LP64(ShouldNotReachHere();)
+#ifdef _LP64
```

Comment:
`#ifndef ... else ...` would look marginally better IMO.

Reply:
Yeah, you're right.  Fixed.